Resources/doc/annotations/csrf.rst

```diff
@@ -0,0 +1,38 @@
+@Security
+=========
+
+.. caution::
+
+    The ``@Csrf`` annotation only works as of Symfony 2.7.
+
+Usage
+-----
+
+The ``@Csrf`` annotation validates csrf-token before executing action::
+
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Csrf;
+
+    class PostController extends Controller
+    {
+        /**
+         * @Csrf("post_delete")
+         */
+        public function deleteAction()
+        {
+            // ...
+        }
+    }
+
+On the template side
+
+    <a href="{{ path('acme_post_delete', {'_token': csrf_token('post_delete')}) }}">delete post</a>
```

Comment:
I would prefer that you use an example that does not use GET as it is really a bad practice that we should not encourage.